docs/.project/RELEASE.md

```diff
@@ -22,45 +24,55 @@ For patch releases turn to [this documentation](./PATCH_RELEASE.md).
-* [ ] verify `develop` is up to date with `master`: `git checkout master && git pull && git checkout develop && git merge master`
 * [ ] smoke test to verify all diagrams can be created
 * [ ] update [Release Info](https://github.com/camunda/camunda-modeler/blob/develop/client/src/plugins/version-info/ReleaseInfo.js)
-* [ ] update CHANGELOG
+* [ ] update [`CHANGELOG`](https://github.com/camunda/camunda-modeler/blob/develop/CHANGELOG.md)
+* [ ] verify `develop` is up to date with `master`: `git checkout master && git pull && git checkout develop && git merge master`
 * [ ] merge to master: `git checkout master && git merge develop`
-* [ ] create release candidate (`npm run release`), e.g. v1.0.0-rc.0
-    * [ ] wait for [GitHub Actions](https://github.com/camunda/camunda-modeler/actions/workflows/RELEASE.yml) to build the executables
+* [ ] create release candidate (`npm run release`), cf. [release schema](https://github.com/bpmn-io/internal-docs/tree/master/release-schema); e.g `v1.0.0-rc.0`
+    * [ ] wait for [Release build](https://github.com/camunda/camunda-modeler/actions/workflows/RELEASE.yml) to create the [artifacts](https://github.com/camunda/camunda-modeler/releases)
 * [ ] execute [integration test](https://github.com/camunda/camunda-modeler/blob/master/docs/.project/INTEGRATION_TEST.md) on [released artifacts](https://github.com/camunda/camunda-modeler/releases)
     * [ ] Works on Linux
     * [ ] Works on Mac
     * [ ] Works on Windows
 * [ ] notify QA about the release so they can test it
-* [ ] write blog post for the [Camunda Blog](https://camunda.com/blog/)
-   * [ ] create a draft post on Wordpress
-   * [ ] send the draft to `Christopher Rogers` to coordinate the publishing
-* [ ] (if the Modeler UI appearance changed) update Camunda Modeler screenshots (and potentially text content) in the docs using [automation scripts](https://github.com/camunda/camunda-docs-modeler-screenshots)
 
 _To be done after integration test was successful or if not issues were fixed._
 
 * [ ] create release (`npm run release`), cf. [release schema](https://github.com/bpmn-io/internal-docs/tree/master/release-schema)
-    * [ ] wait for [GitHub Actions](https://github.com/camunda/camunda-modeler/actions/workflows/RELEASE.yml) to build the executables
-* [ ] send all information required to compile the [release notice email](https://github.com/bpmn-io/internal-docs/blob/master/camunda-modeler/README.md#release-notice-email) to the [`@product-release-presentation-dri` role](https://confluence.camunda.com/pages/viewpage.action?spaceKey=camBPM&title=Release+Presentation+Organization)
+    * [ ] wait for [Release build](https://github.com/camunda/camunda-modeler/actions/workflows/RELEASE.yml) to create the [artifacts](https://github.com/camunda/camunda-modeler/releases)
+
+_To be done once the release is built._
+
+* [ ] write blog post for the [Camunda Blog](https://camunda.com/blog/)
+   * [ ] create a draft post on Wordpress
```

Comment:
Could we link wordpress login here? I always need to search it 

```suggestion
   * [ ] create a draft post on [Wordpress](https://camunda.com/camundalogin)
```

Reply:
We could. However I believe we renamed it as a security precaution.